### PR TITLE
increases plasma consumption in radiation collectors from 0.001 moles to 0.01 moles

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

increase plasma consumption in radiation collectors by 1000% or whatever so their fullness actually MATTERS
with a 0.01 mole usage per process (generally occur every 2 seconds according to round timer) a radiation collector will have burnt through 10 moles of plasma or ~303 kpa in just over 30 minutes, with filled plasma tanks lasting roughly 90 minutes before needing to be refilled or replaced, with cooled tanks being able to hold more plasma and therefore lasting even longer.
all plasma consumed by this is transformed into tritium which can be mixed with oxygen in a rad collector to make research points as if that even matters. The consumption of trit/oxygen has not been changed.

number might be a bit high

### Why is this change good for the game?
more engine maintenance = more for engineers to do when not sticking thumb in ass
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
filling plasma tanks for rad collectors matters now

### What should players be aware of when it comes to the changes your PR is implementing?
the amount of plasma in a rad collector will actually matter, meaning more filled tanks being used in collectors = less needing to refill them
### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
engineering

# Changelog

:cl:  
tweak: increased plasma to tritium rate in radiation collectors from 0.001 to 0.01
/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
this MIGHT cause the rad collectors to eject their tanks directly into the singularity if they are fully emptied so don't
let them do that

**Comment by Yogbot-13** [comment]
#### Test Merge Deployed

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 17047cdcde8e5b32249d1b7d13d0790943c2ba58
Server: 576b980d7860d3a2d8af75956984af14a64580ef

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 17047cdcde8e5b32249d1b7d13d0790943c2ba58
Server: ce13dee3d6e20b89dc7daedfb859ee58550b7dcf

**Comment by MegaEmpirical** [comment]
I like it. Wouldn't mind it being faster either.

Anyway we can read the current mix in the collectors? Use an analyzer on them?

**Comment by Cartlord** [comment]
I find the only issue here is that a lot of people don't even consider that the collectors might be empty - I had a round where the power went out, EVERYONE thought it was some insane sabotage, and then it turned out the collectors were just empty. Could maybe use some signposting, especially considering that (as I see it) many Engineers leave the radiation shields down, which blocks them from seeing the rad collectors at all unless they directly go & check them.

**Comment by MegaEmpirical** [comment]
Exactly, they will learn. Trial by fire. Same any time a new engine is made or some other change.
